client/src/components/Signup.js

```diff
@@ -438,6 +439,15 @@ export default class Signup extends React.Component {
     }
 
     async UNSAFE_componentWillMount() {
```

Comment:
why the use of `UNSAFE_componentWillMount`? I looked it up to see the diff between the UNSAFE version and the other version and it appears this one is legacy code.